safehttp/default_dispatcher.go

```diff
@@ -62,13 +41,15 @@ func (DefaultDispatcher) ContentType(resp Response) (string, error) {
 func (DefaultDispatcher) Write(rw http.ResponseWriter, resp Response) error {
```

Comment:
Please add something to the doc that says it sets CT. I know it's also in the interface definition but I'd like to make sure this is smth that can't just be overlooked by readers.

Reply:
Done.